flink-python/docs/reference/pyflink.datastream/checkpoint.rst

```diff
@@ -81,7 +81,7 @@ The default limit of concurrently happening checkpoints: one.
     CheckpointConfig.set_checkpoint_storage
     CheckpointConfig.set_checkpoint_storage_dir
     CheckpointConfig.get_checkpoint_storage
-    ExternalizedCheckpointCleanup
+    ExternalizedCheckpointRetention
```

Comment:
Also modify above `CheckpointConfig.set_externalized_checkpoint_cleanup` and 
    `CheckpointConfig.is_externalized_checkpoints_enabled` ?

Reply:
Is `CheckpointConfig.is_externalized_checkpoints_enabled` or `CheckpointConfig.get_externalized_checkpoint_cleanup` ？

Reply:
Yeah, I think it's better to since we have modified their parameters.
@HuangXingBo WDYT? And Do we have other things to do about breaking changes in python codes ?

Reply:
Wait a sec, I think it is better to perform a deprecation on `ExternalizedCheckpointCleanup` in python. WDYT?

Reply:
In python, the class name is currently changed directly. Do we need to keep it consistent with Java and create a new `ExternalizedCheckpointRetention.py`?

Reply:
BTW, Let's also extract python related code into one commit which could help us review.
The Java side LGTM. After following the deprecation on python, I think we could merge it.